docs/source/serialization.rst

```diff
@@ -48,6 +48,7 @@ Ready-made configurations include the following models:
 - GPT Neo
 - LayoutLM
 - Longformer
+- Marian
```

Comment:
I'm not sure whether `.rst` files are still allowed with the new `.mdx` doc - does this need updating / changing?

Reply:
Letting @LysandreJik answering this one.

Reply:
I saw the Sylvain recently converted all the RST files to MDX, so I'll rebase and this file should disappear :)